src/pass.h

```diff
@@ -443,6 +444,8 @@ class WalkerPass : public Pass, public WalkerType {
   void setPassRunner(PassRunner* runner_) { runner = runner_; }
 };
 
+std::ostream& printStackIR(std::ostream& o, Module* module);
```

Comment:
This doesn't seem like the right place for this - it doesn't fit in with anything around it, AFAICT.

I'm not sure offhand what the right place is, but I'd look for similar functions and where they are defined.

Reply:
Yes, the location of this function is really inappropriate.

But its definition will be used by the C API, and it will also access `GenerateStackIR` and `PrintStackIR`, I can't find a better place for it.

Hope you can find a more suitable location for it.

Reply:
Hmm, how about at the very end of `wasm-stack.h`? We have similar things at the very end of `wasm.h` for non-stack IR, for comparison.

Reply:
Good idea! I have moved printStackIR's declaration into `wasm-stack.h`.